src/makeRobotAccountant.js

```diff
@@ -6,7 +6,13 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 1;
+
+  return x => {
+    count++;
+
+    return y => (count > 3 && count % 2 !== 0) ? 'Bzzz... Error!' : x + y;
```

Comment:
it's better not combine logical operators with a ternary one, this way code looks messy
use if statement

Reply:
fixed